slave/components/quality_controller/adapter_trimming.go

```diff
@@ -39,7 +39,7 @@ func NewAdapterTrimming(ctx context.Context, dockerCli *client.Client, config *c
 func (a *adapterTrimming) Process() (string, error) {
 
 	// TrimmedFileName is the filename where the trimmed reads are going to be stored.
```

Comment:
Minor but should change the comment capitalization of the variable name as well 🤷‍♂️

Reply:
The variable capitalization matches the comment, does it not?

<img width="155" alt="image" src="https://user-images.githubusercontent.com/19979068/83958790-2376ad00-a833-11ea-9366-a21a3c9642d7.png">

Reply:
This PR has:
```go
// TrimmedFileName is the filename where the trimmed reads are going to be stored.
trimmedOutuptFileName := "trimmed.fastq"
``